packages/node_modules/@ciscospark/redux-module-conversation/src/actions.js

```diff
@@ -114,7 +114,7 @@ export function createConversationWithUser(userId, spark) {
       participants: [userId]
     }, {
       latestActivity: true,
-      activitiesLimit: 30,
+      activitiesLimit: 100,
```

Comment:
Needed to up the limit because if `updates` activity items are the only ones available, there is no way to scroll up to trigger the list to load more history.